Problem2/src/problem2/Problem2.java

```diff
@@ -1,16 +1,68 @@
 package problem2;
-
-/**
- *
- * @author cgallinaro
- */
+import java.util.Scanner;
 public class Problem2 {
-
-    /**
-     * @param args the command line arguments
-     */
     public static void main(String[] args) {
-        // TODO code application logic here
+        Scanner number = new Scanner(System.in);
+        System.out.println("Enter the first grade");
+        double n1 = number.nextDouble();
+        String d1 = grades(n1);
+        System.out.println(d1);
+        System.out.println("Enter the second grade");
+        double n2 = number.nextDouble();
+        String d2 = grades(n2);
+        System.out.println(d2);
+        System.out.println("Enter the third grade");
+        double n3 = number.nextDouble();
+        String d3 = grades(n3);
+        System.out.println(d3);
+        System.out.println("Enter the fourth grade");
+        double n4 = number.nextDouble();
+        String d4 = grades(n4);
+        System.out.println(d4);
+        System.out.println("Enter the fifth grade");
+        double n5 = number.nextDouble();
+        String d5 = grades(n5);
+        System.out.println(d5);
+    }
+        public static String grades(double i){
```

Comment:
Good use of a helper method.  Watch your formatting though (keep your indentations consistent).